scripts/zones/Horlais_Peak/IDs.lua

```diff
@@ -28,8 +28,6 @@ zones[dsp.zone.HORLAIS_PEAK] =
     },
     mob =
     {
-        HELLTAIL_HARRY_OFFSET = 17346645,
-        FIGHTING_SHEEP_OFFSET = 17346684,
```

Comment:
Were these ever used? I can't seem to locate their purpose in the commit history.

Reply:
(I know they aren't now, thats why you are gutting them. wandering wtf they were doing there in the first place tho)

Reply:
https://github.com/DarkstarProject/darkstar/commit/452de08317343a0a7537cac7acfdc50330af3b55

Reply:
ty @zynjec